charts/secrets-store-csi-driver/Chart.yaml

```diff
@@ -9,3 +9,4 @@ sources:
 maintainers:
   - name: Rita Zhang
     email: ritazh@microsoft.com
+apiVersion: v1
```

Comment:
Added this to prevent `helm lint` errors. Do you want to bump the chart version as well?

Reply:
Bumping it to `v2` will mean the chart won't work with helm2. I think it's acceptable for now to keep it at `v1` so that the chart works with helm2 and helm3. If we do introduce any helm3 specific change in the future or when helm2 is deprecated, this can be bumped up.

Reply:
nit: can you move the apiVersion to the first line in `Chart.yaml`?

Reply:
I meant bump the chart version to `0.0.10`. Not the `apiVersion`. I agree that should be v1 unless required.

Reply:
Also moved line.